passkey-client/Cargo.toml

```diff
@@ -14,21 +14,30 @@ categories.workspace = true
 [lints]
 workspace = true
 
+[features]
+tokio = ["dep:tokio"]
+testable = ["dep:mockall"]
+
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-passkey-authenticator = { path = "../passkey-authenticator", version = "0.2" }
+passkey-authenticator = { path = "../passkey-authenticator", version = "0.3" }
 passkey-types = { path = "../passkey-types", version = "0.2" }
 public-suffix = { path = "../public-suffix", version = "0.1" }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 ciborium = "0.2"
+mockall = { version = "0.11", optional = true }
```

Comment:
**Issue:** Don't forget to add mockall as a `dev-dependency` as well. Currently this branch is broken if you run `cargo test -p passkey-client`